src/services/middlewareServices/AuthenticationMiddlewareService.ts

```diff
@@ -42,7 +49,16 @@ class AuthMiddlewareService {
     return true
   }
 
-  verifyJwt({ cookies, url }) {
+  verifyJwt({
+    cookies,
+    url,
+  }: {
+    cookies: {
+      isomercms: string
+      isomercmsE2E?: string
+    }
+    url: string
```

Comment:
if we do simplify the type above, we can use it here too hehe

Reply:
https://github.com/isomerpages/isomercms-backend/pull/479/commits/55ed491e445f3f25993036851c1417bf95f18a96